webapp/src/pages/boardPage.tsx

```diff
@@ -163,13 +163,20 @@ const BoardPage = (props: Props): JSX.Element => {
         }
     }, [board?.title, activeView?.title])
 
+    if (props.readonly) {
+        useEffect(() => {
+            if (board?.id && activeView?.id) {
+                TelemetryClient.trackEvent(TelemetryCategory, TelemetryActions.ViewSharedBoard, {board: board?.id, view: activeView?.id})
+            }
+        }, [board?.id, activeView?.id])
```

Comment:
Nice optimization.  I guess telemetry was tracked every render before this?

Reply:
It was actually being tracked once unless the `workspaceId` changed, which would miss view updates